docs/configuration/monitoring.md

```diff
@@ -12,32 +12,51 @@ we have all functionality available on Grafana.
 
 The simplest way to setup prometheus in your cluster is by using the
 [Prometheus-Operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
-bundle.
+bundle. Start by adding the custom resources required for installing this bundle in your cluster.
 
 ```sh
+# NOTE: if installing on microk8s use `microk8s.kubectl` instead of `kubectl`
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml
+```
+
+Obtain the configurations files for [loki](monitoring-conf/loki-values.yaml),
+[promtail](monitoring-conf/promtail-values.yaml), and
+[prometheus-operator](monitoring-conf/prometheus-operator-values.yaml) and place them
+in the same folder where you'll run the helm commands.
+
+**You must edit:**
 
-sleep 5 # wait for crds to be created
+-   the Grafana password for the `admin` user in `prometheus-operator-values.yaml`
+-   the storage class for loki persistence in `loki-values.yaml`
 
-helm upgrade --install monitoring stable/prometheus-operator \
-  --version 6.9.3 \
-  --namespace monitoring \
-  --set prometheusOperator.createCustomResource=false \
-  --set grafana.service.type=NodePort\
-  --set grafana.adminPassword="CHANGE_HERE"
+You can now install the monitoring bundle on your cluster:
+
+```sh
+helm repo add loki https://grafana.github.io/loki/charts
 
-kubectl get svc monitoring-grafana -n monitoring
+kubectl create namespace monitoring
+
+helm upgrade --install --atomic loki loki/loki --version 0.17.0 \
+  --recreate-pods --namespace monitoring --values loki-values.yaml
+
+helm upgrade --install --atomic promtail loki/promtail --version 0.13.0 \
+  --recreate-pods --namespace monitoring --values promtail-values.yaml
+
+helm upgrade --install --atomic monitoring stable/prometheus-operator --version 6.9.3 \
+  --recreate-pods --namespace monitoring --values prometheus-operator-values.yaml
 ```
 
-Now that you have prometheus and grafana you can enable `ServiceMonitors` and `GrafanaDashboards`
-for Codacy components:
+Now that you have prometheus and grafana installed you can enable `serviceMonitors`
```

Comment:
Just a nitpick, but check the capitalization of the 3rd party tools whenever is makes sense (when we're not referring to a lowercase command or setting, for example):

```suggestion
Now that you have Prometheus and Grafana installed you can enable `serviceMonitors`
```